websites/K/KaniNetwork/presence.ts

```diff
@@ -2,47 +2,55 @@ const presence = new Presence({
   clientId: "801742167608787015"
 });
 
+function LookingKaniShiel(presenceData: PresenceData) {
+  presenceData.smallImageKey = "logo";
+  presenceData.smallImageText = "KaniShiel";
+}
```

Comment:
I literally told you to remvoe it completely, since you will have 2 times same logo on largeImage and smallImage. Maybe if you will have something else to put as smallImage key, you will be able but now, no

Reply:
Ah sorry, I didn't understanded this ^^'